examples/doxygen/block_matrix_array.cc

```diff
@@ -13,57 +13,38 @@
 //
 // ---------------------------------------------------------------------
 
-
 // See documentation of BlockMatrixArray for documentation of this example
 
 #include <deal.II/base/logstream.h>
 #include <deal.II/lac/block_matrix_array.h>
-#include <deal.II/lac/full_matrix.h>
-#include <deal.II/lac/vector.h>
 #include <deal.II/lac/block_vector.h>
+#include <deal.II/lac/full_matrix.h>
 #include <deal.II/lac/precondition.h>
 #include <deal.II/lac/solver_cg.h>
 #include <deal.II/lac/solver_gmres.h>
+#include <deal.II/lac/vector.h>
 
-#include <iostream>
 #include <fstream>
+#include <iostream>
 
 using namespace dealii;
 
-double Adata[] =
-{
-  4., .5, .1, 0.,
-  .5, 4., .5, .1,
-  .1, .5, 4., .5,
-  0., .1, .5, 4.
-};
+double Adata[]
+  = {4., .5, .1, 0., .5, 4., .5, .1, .1, .5, 4., .5, 0., .1, .5, 4.};
```

Comment:
`clang-format` can't handle this. We should use a multidimensional array instead.

Reply:
Either that, or wrap such rare cases into
```
/* clang-format off */
...
/* clang-format on */
```